src/pages/Landing/Header.tsx

```diff
@@ -0,0 +1,30 @@
+import cellBg from "../../assets/cell.png";
+
+const Header = () => {
+  return (
+    <header
+      style={{
+        backgroundImage: `linear-gradient(to bottom, #2C3333 7%, rgba(0, 0, 0, 0) 46%, rgba(0, 0, 0, 0) 96%, #2C3333), url(${cellBg})`,
+        backgroundPosition: "center",
+        backgroundSize: "auto, 45px",
+      }}
+    >
+      <div className="container flex flex-col justify-center items-center gap-8 min-h-screen max-w-5xl text-center">
+        <h1>
+          Streamline Your Software Development with Our Cutting-Edge{" "}
+          <span className="text-accent">CI/CD Tool</span>
+        </h1>
```

Comment:
can you reduce the size of this header as it is overflowing in small sized screens.
![image](https://github.com/catalyst-ci-cd/catalyst-frontend/assets/52636794/7934d796-6f61-4d39-9715-e015243991e5)